proposal.rst

```diff
@@ -180,39 +180,57 @@ Motivations
 
 **Why have you chosen this particular project?**
 
-It is interesting to me to benchmark ranking algorithms on big databases.
+Information Retrieval is interesting to me as application of statistics. This project allows me to broaden horizons with practical examples. Also programming practice and communicating with mentors are very important to me.
 
 **Who will benefit from your project and in what ways?**
 
 .. For example, think about the likely user-base, what they currently have to
 .. do and how your project will improve things for them.
 
-Project I want to do assumes that there will be tool for evaluating Letor on different datasets and compare its performance, so it useful for users wich want to understand Xapian abilities.
+Project I want to do assumes that there will be tool for evaluating Letor on different datasets and compare its performance. If it is completed successfully, there free suite with several ranking algorithms and features will available for users. It is useful not only for future developers, but for participants IR contests like TREC. Also it is possible to implement automated configuration choose using score classes.  
 
 Project Details
 ---------------
 
-.. Please go into plenty of detail in this section.
-
 **Describe any existing work and concepts on which your project is based.**
 
-There are several ranking algorithms with pretty same api implemented in Letor. So I think manager class should be developed to evaluate ranking and logging results. It will better to integrate it with testing classes, because it makes possible to develop suite for perfomance rating, wich will useful not only for ranking, but for all test cases. 
+There are several ranking algorithms with pretty same api implemented in Letor. So I think manager class should be developed to evaluate ranking and logging results. It will better to integrate it with Letor classes. Additional task is to improve perftest suite by running it on real data, wich will useful not only for ranking benchmark, but for all test cases. More detailed:
```

Comment:
I think you should try to spell out in more detail what you mean by a manager class to evaluate ranking and logging results. That sounds like two different pieces of work, so it's not immediately clear why one manager would deal with this.

Evaluation (how well we rank documents for queries) and performance testing are very different things. If you want to tackle them both, then I'd treat them as totally different subprojects, which means they need their own rationale, their own description and project breakdown, and their own steps in the timeline.